employee_project/employee_class.py

```diff
@@ -0,0 +1,215 @@
+
+import os
+import re
+from Constants import *
+emp_id = 0
+employee_database = dict()
+
+
+class Employee_info():
+    """
+    My Employee_info class works get the employee details and add the employee details and delete the employee details and
```

Comment:
Change the doc sting of the class, Employee_info class handled the all the employee related functionalities.